recovery/multiimagewritethread.cpp

```diff
@@ -530,7 +530,13 @@ bool MultiImageWriteThread::processImage(OsInfo *image)
     QVariantList vpartitions;
     foreach (PartitionInfo *p, *partitions)
     {
-        vpartitions.append(p->partitionDevice());
+        QString part = p->partitionDevice();
+        if (part.left(13) != "/dev/mmcblk0p")
```

Comment:
Just out of curiosity, what happens if you remove this special-casing for `/dev/mmcblk0` and instead _always_ use the partuuid?

Reply:
We used to have some Linux distributions that came with an initramfs that did not support partuuid parameters (e.g. OSMC).
Not sure if that's still the case in recent versions though.

Reply:
@procount I wonder if it'd be worth adding an extra param to the `os.json` / `os_list.json` files stipulating whether they support partuuid parameters, and then only using this partuuid code if they do?

Reply:
Can we piggy-back the `usb_root` or `usb_boot` parameters for that? I think they serve a similar purpose...?

Reply:
IIRC, if an OS doesn't support partuuid, then it can't be installed to USB. If so, then the additional code won't be necessary. I'm sure @XECDesign  explained the use of `supports_usb_root` and `supports_usb_boot` somewhere but I can't find the reference.